ydb/core/blobstorage/ut_vdisk/lib/astest.h

```diff
@@ -82,7 +82,6 @@ inline void TTestWithActorSystem::Run(NActors::IActor *testActor) {
         NKikimrServices::EServiceKikimr_Name
     );
     TString explanation;
-    logSettings->SetLevel(NLog::PRI_TRACE, NActorsServices::EServiceCommon::HTTP, explanation);
```

Comment:
just something I mistakenly introduced in the previous commits

Reply:
don't need excuses